go-apps/meep-gis-engine/api/swagger.yaml

```diff
@@ -283,6 +350,50 @@ definitions:
     example:
       active: true
       type: "MOBILITY"
+  DistanceParameters:
+    type: "object"
+    properties:
+      assetName:
+        type: "string"
+        description: "Asset name of a second element for query purpose. If present,\
+          \ latitude and longitude are ignored."
+      latitude:
+        type: "number"
+        format: "float"
+        description: "Latitude of a second element for query purpose. Taken into account\
+          \ only if AssetName is not present."
+      longitude:
+        type: "number"
+        format: "float"
+        description: "Longitude of a second element for query purpose. Taken into\
+          \ account only if AssetName is not present."
+    description: "Parameters for distance query purpose."
+    example:
+      latitude: 0.8008282
+      assetName: "assetName"
+      longitude: 6.0274563
+  DistanceResponse:
```

Comment:
We could call this object simple _Distance_. i.e. it would return a _Distance_ object.

Reply:
removed the Response part of the object name